types/k6/experimental/timers.d.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ *  This module provides setInterval, setTimeout and co.
+ * Not to be used mostly for testing purposes.
```

Comment:
```suggestion
```

I think this comment should probably be removed now that we are provinding them with the idea of people using them ;)